test/DurableTask.SqlServer.AzureFunctions.Tests/MultiTenancyCoreScenarios.cs

```diff
@@ -0,0 +1,41 @@
+namespace DurableTask.SqlServer.AzureFunctions.Tests
```

Comment:
Please add the common copyright header to the top of this new file.

Reply:
Done